config/application.rb

```diff
@@ -80,5 +80,9 @@ class Application < Rails::Application
     # redis-rails does not support cache versioning
     config.active_record.cache_versioning = false
     config.active_record.collection_cache_versioning = false
+
+    # Setting enables YJIT as of Ruby 3.3, to bring sizeable performance improvements. We are
+    # deploying to a memory constrained environment so we set this to `false`.
+    Rails.application.config.yjit = false
```

Comment:
```suggestion
    config.yjit = false
```
based on the context, probably this?